process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,87 +2,118 @@
 title: DocuSign connector
 ---
 
-The DocuSign connector is used to send documents via email to be digitally signed. The process flow waits for a document to be signed before continuing with the process. The signed document is saved to the Content Services repository. Multiple signers is supported, which means more than one name, email address and signature, can be used.
+The DocuSign connector provides a standard mechanism in Alfresco Process Automation to send documents for digital signing in
+[DocuSign](https://www.docusign.com){:target="_blank"}.
+
+The DocuSign connector is displayed on the process diagram as a pen.
 
 > **Important**: The DocuSign connector requires a [DocuSign](https://www.docusign.com/){:target="_blank"} account to handle document signing. This account is separate to the Alfresco hosted environment and should be created and managed by customers.
 
-The DocuSign connector is displayed on the process diagram as a pen.
+The actions that can be executed using the DocuSign connector are:
+
+* [SEND_FOR_SIGNATURE](#send_for_signature)
+* [DOWNLOAD_DOCUMENT](#download_document)
+
+As part of the BPMN definition process, any service task that is responsible for sending the document needs to set the `docusignconnector.SEND_FOR_SIGNATURE` or `docusignconnector.DOWNLOAD_DOCUMENT` properties as the value for its implementation attribute.
+
+The following input parameters must also be provided for the DocuSign API in the Service task depending on the implementation.
 
-## Sign document
+## SEND_FOR_SIGNATURE
 
-The **SIGNDOCUMENT** action is used by the DocuSign connector to request a digital signature on a document.
+The **SEND_FOR_SIGNATURE** action is used by the DocuSign connector to request a digital signature on a document.
 
-The input parameters to request a DocuSign signature are:
+The input parameters for SEND_FOR_SIGNATURE are:
+
+**Note:** The connector can only receive either **signers** or **recipientEmail** but not both at the same time.
```

Comment:
What do you think if we move the note below the inputs, so the user will have context of each of them?

Reply:
Changed